Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /opt/app
 # Copy basic files for installing dependencies
 COPY tsconfig.json package.json package-lock.json /opt/app/
 COPY src /opt/app/src
+COPY config /opt/app/config
```

Comment:
This is a problem, we should not be loading multiple configs into the dockerfile.

I.e. this goes back to my other comment about only having a single config file.

Otherwise, we could end up in a situation where the wrong/incorrect config file may be accidentally loaded.

I don't want this to happen, I only want a single source of config truth at all times.

Reply:
Agreed, changed accordingly